src/os/ui/window.js

```diff
@@ -660,7 +661,10 @@ os.ui.WindowCtrl = function($scope, $element, $timeout) {
     $scope['x'] = (container.width() - $scope['width']) / 2;
   }
 
-  if ($scope['y'] == 'center') {
+  if ($scope['bottom'] && $scope['bottom'] != 'auto') {
```

Comment:
This check (and all the subsequent checks below) for `$scope['bottom']` do not properly handle the case where `bottom` is `0`, which is falsey. I think you want `$scope['bottom'] != null` instead.

Reply:
good catch. fixed.